src/material/icon/icon.md

```diff
@@ -27,7 +27,7 @@ use the desired font, or to an alias previously registered with
 
 Fonts can also display icons by defining a CSS class for each icon glyph, which typically uses a
 `:before` selector to cause the icon to appear.
-[FontAwesome](https://fortawesome.github.io/Font-Awesome/examples/) uses this approach to display
+[FontAwesome](https://github.com/FortAwesome/angular-fontawesome) uses this approach to display
```

Comment:
I'm not sure that this is the correct link since it points to FontAwesome's own Angular component. I think that the old link was showing some examples of the actual icons. Maybe we should link to this instead? https://fontawesome.com/icons

Reply:
I understand your point @crisbeto, to be honest I don't know what the broken link was showing, if was just icons examples or implementations examples.
In my situation I was looking for examples of how to start use the FontAwesome and not just the icon itself, than the github angular-fontawesome helped me in the both situation, how to do de integration and also where I can check the icons. This is the reason that I suggested to use this link and not just the icons.

Reply:
It definitely makes sense not to link to a 404, but the whole point of the doc is to show how to use a font icon set with `mat-icon`.

Reply:
Indeed should be the [example link](https://fontawesome.com/icons), you are right @crisbeto.
I will change and do the commit again.

In this case, and seeing that the Fontawesome is also an official Angular component for icons, is it possible to create a session in the documentation to explain how to install and use the Font Awesome in the project?

Reply:
I don't think that we should mention the Font Awesome component, because we aren't the ones maintaining it.